prometheus/generic/setup-bind-exporter/tasks/prereqs.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Ensure epel-release is installed
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - epel-release
+
+- name: Ensure pip is installed
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - python-pip
```

Comment:
Can these be combined into one task?

Reply:
I tried, but apparently python-pip is dependent on epel-release. Therefore the task fails "No package matching 'python-pip' found" if these are in single loop.

Reply:
Thank you for looking into it, I should've verified this before asking.